src/Microsoft.AspNet.Authentication.OAuth/Notifications/OAuthAuthenticatedContext.cs

```diff
@@ -39,11 +36,6 @@ public class OAuthAuthenticatedContext : BaseContext<OAuthAuthenticationOptions>
         }
 
         /// <summary>
-        /// Gets the JSON-serialized user.
-        /// </summary>
-        public JObject User { get; protected set; }
```

Comment:
If we're arguing semantics it may not be a user at all, it could be the result of machine to machine, and the round trip gives you machine permissions ...

I wouldn't want it in the base class though, as that's unrealistic. It indicates there will be a back channel communication to populate it in all cases.